ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -27,6 +27,12 @@ distro_custom() {
     if ! rpm -q nfs-utils; then
         dnf -y install nfs-utils
     fi
+
+    # CORCI-1096
+    sed -e 's/^\(hostname *= *\)[^ ].*$/\1 mail.wolf.hpdd.intel.com:25/' < /usr/share/doc/esmtp/sample.esmtprc > /etc/esmtprc
```

Comment:
(style)  line over 80 characters

Reply:
Temporary change to be removed when CORCI-1096 is complete.